mmv1/templates/terraform/constants/artifact_registry_repository.go.tmpl

```diff
@@ -43,3 +43,22 @@ func upstreamPoliciesDiffSuppress(k, old, new string, d *schema.ResourceData) bo
   newSet := schema.NewSet(schema.HashString, newHashes)
   return oldSet.Equal(newSet)
 }
+
+func isDefaultEnum(val any) bool {
```

Comment:
where is the function being called?

Reply:
It is used in the return value of the function emptyMavenConfigDiffSuppress below.